actors/builtin/miner/miner_actor.go

```diff
@@ -689,7 +689,7 @@ func (a Actor) PreCommitSectorBatch(rt Runtime, params *PreCommitSectorBatchPara
 		}
 		sectorNumbers.Set(uint64(precommit.SectorNumber))
 
-		if !CanPreCommitSealProof(precommit.SealProof, nv) {
+		if !CanPreCommitSealProof(precommit.SealProof) {
 			rt.Abortf(exitcode.ErrIllegalArgument, "unsupported seal proof type %v at network version %v", precommit.SealProof, nv)
```

Comment:
suggestion: the only other use of `nv` is in this error message.  I think its safe to remove this part of the error message and the local variable.